swaScraper.py

```diff
@@ -159,39 +165,54 @@ def scrape(args):
 
         if args.seniors:
             outbound_fares = browser.find_elements_by_tag_name("ul")[1]
+        elif args.flightnum:
+            # find li which flight number exists
+            xpath = '//button[contains(@class,\'flight-number\') and contains(.,"{0}")]/ancestor::*[self::li][1]'.format(args.flightnum)
+            outbound_fares = browser.find_element_by_xpath(xpath)
         else:
             outbound_fares = browser.find_elements_by_tag_name("ul")[2]
 
-        outbound_prices = outbound_fares.find_elements_by_class_name("currency_dollars")
+        if args.points:
```

Comment:
Can you think of a way of re-writing this without the redundancy?

Reply:
How about specify the replace character as a variable ("$" for dollars and "," for points?) and search item "currency_dollars" or "currency_points" as well, then do this once?

Reply:
Yup, sounds good to me.

>if args.points:
>	delimiter = ','
>	outbound_prices = outbound_fares.find_elements_by_class_name("currency_points")
>else:
>	delimiter = '$'
>	outbound_prices = outbound_fares.find_elements_by_class_name("currency_dollars")>

>for price in outbound_prices:
>		realprice = price.text.replace(delimiter, "")
>		outbound_array.append(int(realprice))

Reply:
I did something similar but a little different.  Check it out and see what you think.